editor/components/post-permalink/index.js

```diff
@@ -112,6 +113,8 @@ class PostPermalink extends Component {
 						href={ getWPAdminURL( 'options-permalink.php' ) }
 						onClick={ this.addVisibilityCheck }
 						target="_blank"
+						icon={ false }
+						rel={ null }
```

Comment:
So, as I recall, setting `rel` as `null` to override default behavior is needed because we don't want the external values to apply for links which open in a new tab but are within the admin? Can we infer this programmatically such that we don't impose this requirement on the developer to understand?

Example: https://github.com/aduth/dones/blob/0450c88429deb4546f72d20f915551ec62a33de3/src/components/link/index.js#L44-L47